src/fs.cpp

```diff
@@ -128,7 +128,7 @@ bool FileLock::TryLock()
     if (hFile == INVALID_HANDLE_VALUE) {
         return false;
     }
-    _OVERLAPPED overlapped = {};
+    _OVERLAPPED overlapped = {0};
```

Comment:
20744 - why that is added? it seems unrelated; moreover useless: there's no UB without `0`, it's completely fine

As I checked, bitcoin doesn't have it

Reply:
Resolved in latest push!